assets/.eslintignore

```diff
@@ -2,4 +2,6 @@ webpack.config.js
 .eslintrc.js
 # This file was generated by Phoenix
 socket.js
-node_modules
\ No newline at end of file
+node_modules
+coverage
+*.json
```

Comment:
It looks like the only reason `json` files would be getting checked in the first place is because we explicitly include `.json` in our ESLint command line in `package.json`. We could remove that, instead of "including and then excluding" it.